### DISCUSSION TITLE
Fazendo as Coisas com Raiva

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#521] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/gifs/fazendo-as-coisas-com-raiva/

**Comment by the-coffee-monk** [comment]
Não entendo o *frenesi* todo com esse filme Internet afora. É tão... chato :neutral\_face:

**Comment by maryon-telvanni** [comment]
> [The Coffee Monk](</perfil/?user=the-coffee-monk>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/gifs/fazendo-as-coisas-com-raiva/#comment_1173907112>):

> Não entendo o *frenesi* todo com esse filme Internet afora. É tão... chato 😐

Também não gostei não
